src/NuGet.Core/NuGet.Packaging/compiler/resources/nuspec.xsd

```diff
@@ -51,7 +51,7 @@
       </xs:sequence>
       <xs:attribute name="targetFramework" type="xs:string" use="required" />
     </xs:complexType>
-  
+
```

Comment:
revert unnecessary changes, these can be cleaned up later.

Reply:
oops! I thought I'd gotten rid of this lol